src/main/scala/io/iohk/ethereum/jsonrpc/server/http/JsonRpcHttpServer.scala

```diff
@@ -50,14 +65,30 @@ trait JsonRpcHttpServer extends Json4sSupport {
     (path("healthcheck") & pathEndOrSingleSlash & get) {
       handleHealthcheck()
     } ~ (pathEndOrSingleSlash & post) {
-      entity(as[JsonRpcRequest]) { request =>
-        handleRequest(request)
+      (extractClientIP & entity(as[JsonRpcRequest])) { (clientAddress, request) =>
+        handleRequest(clientAddress, request)
       } ~ entity(as[Seq[JsonRpcRequest]]) { request =>
         handleBatchRequest(request)
       }
     }
   }
 
+  def handleRequest(clientAddress: RemoteAddress, request: JsonRpcRequest): StandardRoute = {
+    if (ipTrackingEnabled && request.method != FaucetJsonRpcController.Status) {
```

Comment:
Why are we only disabling this for the faucet status? Maybe it's because it's used as the healthcheck? I though the healthcheck was something separately

Reply:
I added a `FIXME` for this, the faucet Status should somehow work as a healthcheck so I left it out of the rate limiting in case it's being used by some of our services. Eventually it should be part of the healthcheck or be handled more elegantly